packages/components/form/__tests__/form.spec.ts

```diff
@@ -363,6 +363,34 @@ describe('Form', () => {
     expect(addressField.validateMessage).toBe('')
   })
 
+  test('scroll to field', () => {
+    const wrapper = mountForm({
+      template: `
+        <div>
+          <el-form ref="form">
+            <el-form-item prop="name" ref="formItem">
+              <el-input></el-input>
+            </el-form-item>
+          </el-form>
+        </div>
+      `,
+      data() {
+        return {
+          form: {
+            name: '',
+          },
+        }
+      },
+    })
+
+    const scrollIntoViewMock = jest.fn()
+    window.HTMLElement.prototype.scrollIntoView = function() { scrollIntoViewMock(this) }
```

Comment:
I think we need to `destroy` the function you mocked for `window.HTMLElement` via this:
```ts
const oldScrollIntoView = window.HTMLElement.prototype.scrollIntoView
window.HTMLElement.prototype.scrollIntoView = function() { scrollIntoViewMock(this) }

// after finished
window.HTMLElement.prototype.scrollIntoView = oldScrollIntoView
```

Reply:
Other than this I don't have any comments.

Reply:
But `window.HTMLElement.prototype.scrollIntoView` is `undefined`

Reply:
It's true, even though it is undefined here because of `jsdom` did not add support for that. If one day we switch to `puppeteer` this could break the entire test.

Reply:
Just for safety.